apps/test/util/testUtils.js

```diff
@@ -110,6 +110,29 @@ export function dragToVisualization(type, left, top) {
   });
   $(document).trigger(drag);
 
+  //
+  // Apply border-box styling that would normally come from applab css files
+  // on the element types: input, div.textArea, button, select, img, label
```

Comment:
This feels like a hack. Why is it only being applied on drag?

Reply:
Our test code has been broken on this front for a while. Not sure if we could provide a better fix to actually run these tests with proper CSS applied?
I spent a long time debugging this and was surprised to see that we aren't really testing it in a "valid" scenario (with the CSS styles applied).
We were lucky before because the padding values were small enough that the grid alignment code rounded things back down and no one noticed.

Reply:
Oh weird.  I wonder if that has anything to do with us still running unit tests on Phantom.

Reply:
Not certain. It still occurs when running with `BROWSER=Chrome` while debugging. I can inspect the elements and see that none of the style sheet classes have been applied.